nuitka/nodes/ContainerMakingNodes.py

```diff
@@ -24,6 +24,7 @@
 from nuitka.PythonVersions import needsSetLiteralReverseInsertion
 
 from .ExpressionBases import ExpressionChildrenHavingBase
+from .IterationHandles import ListAndTupleContainerMakingIterationHandle
```

Comment:
Did you consider putting them next to the node? I am not sure, but my suspect is their are so close friends, they probably need to be looked at as one thing very often.

Reply:
Do you mean next to `NodeMakingHelpers` ?

Reply:
No, more generally I think of a layout, where related things stay together.

`nuitka.nodes.IterationNode.IterationHandleOfSomething`
`nuitka.nodes.Somethings.SomethingNode`

is a bit tearing apart knowledge about `SomethingNode`. Because the iteration handle is going to be deeply tied to it, I would prefer this layout:

`nuitka.nodes.Somethings.IterationHandleOfSomething`
`nuitka.nodes.Somethings.SomethingNode`

In this, related information is dealt with in one place. The iteration handle details are private and result of being huge friends to the node type.

Only base classes, or relatively generic stuff should go into the `IterationHandles` module, and generic stuff needs no private details and is used by multiple things.

With the range nodes, I think it's very evident, that they are something I want to look at the same time as its iteration handles.

For now, leave them where they are. I just wanted to make sure you considered the question, and I want to discuss with you the distribution of responsibility between iteration handle and node separately, and that is going to affect such decisions a lot.